src/app/containers/RadioPageBlocks/Blocks/Datestamp/__snapshots__/index.test.jsx.snap

```diff
@@ -0,0 +1,35 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`AudioPlayer blocks Datestamp should render correctly 1`] = `
+.c0 {
+  font-size: 0.9375rem;
+  line-height: 1.25rem;
+  font-family: ReithSans,Helvetica,Arial,sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  color: #3F3F42;
+  padding-bottom: 1.5rem;
+  margin: 0;
+}
+
+@media (min-width:20rem) and (max-width:37.4375rem) {
+  .c0 {
+    font-size: 1rem;
+    line-height: 1.375rem;
+  }
+}
+
+@media (min-width:37.5rem) {
+  .c0 {
+    font-size: 1rem;
+    line-height: 1.375rem;
+  }
+}
+
+<p
```

Comment:
Please can we update to a span? This is going inside the h1, ya?

Reply:
Just leaving this note for future visitors - following conversation with Catharine the a11y impact on this component & the brand title is considerable and is now captured in this issue https://github.com/bbc/simorgh/issues/6425